apps/web/src/hooks/useMetadata.ts

```diff
@@ -1,46 +1,36 @@
 'use client'
 
-import { useEffect, useRef, useState } from 'react'
+import { useEffect, useState } from 'react'
 
 import type { ConversationMetadata } from 'promptl-ai'
 import { useDebouncedCallback } from 'use-debounce'
 
 import type { ReadMetadataWorkerProps } from '../workers/readMetadata'
 
+const worker =
```

Comment:
I did worker at module level. Does it need to be recreated each time the hook is used?

Reply:
This was something I was checking before finding the other bug. This is not a bug per se but I think is better this way.